### DISCUSSION TITLE
Multiple Fleet nodes?

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#1853] [Unsupported Versions] [answered]
Hi, can a deployment have multiple Fleet nodes?

**Comment by defensivedepth** [comment, marked as answer]
I have updated our documentation around this: https://docs.securityonion.net/en/2.3/architecture.html#fleet-standalone-node

Currently a deployment can only have one Fleet node.

What are you looking to do?

**Reply by OzzyKampha** [reply]
Thinking about having multiple separate networks with endpoints in each. It would be nice to have a node on each network and have that single node to connect to the master that is behind firewalls. So you dont need to open for every endpoint
